### DISCUSSION TITLE
Organic supports

### DISCUSSION THREAD

**Discussion by inline6power** [supermerill/SuperSlicer#4106] [General]
Just wanted to ask if this will be coming with updates? I am not in a hurry or asking for a timeframe just want to know if they will be added to superslicer. I am currently using orca for this but super slicer just flat out prints and works better than anything out there. Only thing I absolutely with super slicer had or has is organic supports

**Comment by supermerill** [comment]
in 2.7 merge

**Reply by gyscos** [reply]
Any news on this 2.7 merge?

**Reply by supermerill** [reply]
I do a dev update every one or two week in discord.
It's improving, it's now in a state where it can print stuff not too badly.
Bugs still appearing every day and so.
I focus on it, so sadly I stopped looking at most of github last month, until I can output an alpha.

**Comment by Gudzonskiy** [comment]
Dear developer of the cool SuperSlicer program! Is it possible for you to add a function or option for slicing code in pendulum mode to SuperSlicer? Example: there are three models on the platform: "1", "2", "3" we cut and they are printed in this sequence - "1", "2", "3", rise along the "Z" axis "3", "2", "1", rise along the Z axis "1", "2", "3", and so on Further! I searched in other popular programs today, but didn’t find them, or didn’t search well! This option will greatly reduce printing time! Sorry I'm not an English speaker so machine translation ;)

**Reply by supermerill** [reply]
hi
You should [open an issue](https://github.com/supermerill/SuperSlicer/issues/new?assignees=&labels=&projects=&template=feature_request.md) (as feature request). 
Please also add more pictures, drawing, schematics, even more if you can't write english.
currently I don't understand what you want, at all.

**Reply by Paradisebaker** [reply]
Hi Merill
I believe his aim is to reduce travel time between the individual parts on the build plate.
The regular way of printing is to keep the same sequence throughout all layers, which could mean that there is a lot of travel.
Lets say you have one piece in the left front quadrant of your buildplate (part A), one in the middle (part B) and another one in the right rear quadrant (part C).
The normal way of printing would be to print one layer of part A, then move to part B and last part C. the next layer would start again with part A, meaning there is a whole lot of empty travel from the right rear (where part C is) to the left front (part A), just to keep the sequence the same.
The way I understand Gudzonskiy is that he wants to print part A, B, C like the original, but then start the next layer with part C (where he is already), then move via part B to part A. next layer would start with A again.
Although this would reduce travel quite a lot, there might be the issue of cooling between layers for the parts that do two layers right after each other (part C at the end of layer 1 and at the beginning of layer 2).
This could work if the objects are big enough or if some minimum layer time would be set.
I believe that Simplify3D did something like this in version 4, but I could be mistaken (too long ago).
Just my 5 cents....

**Reply by Paradisebaker** [reply]
I just doublechecked my old S3D installation and this seems to have been the default behaviour.

**Reply by LaserBee** [reply]
Another neat thing would be if you could tell it to do, say three layers on each one before moving to the next. Not enough to get in the way of your printer while it's doing another piece, but it would mean three times less travel in total. Like "A-A-A-B-B-B-C-C-C". You'd still only have built up 1mm (assuming 0.3mm layers) on each piece so shouldn't get in the way of anything...

EDIT: Might have to increase z-hop a wee bit though.

**Comment by Gudzonskiy** [comment]
Paradisebaker got me right!
